lib/Form/Util/Styles.php

```diff
@@ -158,7 +158,19 @@ public function renderFormSettingsStyles(array $form, string $selector, string $
     $formWrapperStyles = $selector . '{' . join(';', $styles) . ';}';
 
     // Media styles
-    $media = "@media (max-width: 500px) {{$selector} {background-image: none;}}";
+    $media = "@media (max-width: 500px) {{$selector} {
+      animation: none;
```

Comment:
The new rules should apply only to slide-in, fixed bar and pop up. With this implementation, it will apply to all forms including widgets and below posts. We need to apply additional selectors for form types. You can use $displayType variable and FormEtity display type constants to implement it.